bin/run-fisher.sh

```diff
@@ -1,21 +1,14 @@
 #!/bin/sh
-
 set -xe
-
-export ds_importer="fisher"
-
-export ds_train_batch_size=32
-export ds_dev_batch_size=32
-export ds_test_batch_size=32
-
-export ds_epochs=20
-export ds_display_step=1
-export ds_validation_step=5
-export ds_checkpoint_step=5
-
 if [ ! -f DeepSpeech.py ]; then
     echo "Please make sure you run this from DeepSpeech's top level directory."
     exit 1
 fi;
 
-python -u DeepSpeech.py
+python -u DeepSpeech.py \
+  --importer fischer \
+  --train_batch_size 32 \
+  --dev_batch_size 32 \
+  --test_batch_size 32 \
+  --learning_rate 0.0001 \
+  "$@"
```

Comment:
The previous values for ds_epochs, ds_display_step, ds_validation_step, and a value for the checkpoint interval should be passed here.